ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -122,6 +153,27 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "code",
+            "source": [
+                "public bool isCorrectValue(double value, bool isProjectedValue) => isProjectedValue? Math.Abs(value) <= BenchmarkPrecision : true;"
```

Comment:
Why do you need isProjectedValue?

Reply:
Because ValueN is not equal 0 for the variables contained in the DataSource.
![image](https://user-images.githubusercontent.com/120557622/223155312-d08533c8-44f4-472c-826f-656a58c5b8b0.png)

Reply:
@dcolleoni Davide, if we don't use isProjectedValue, we would need to implement another logic, which would require either an additional method that does the same check or use of reflections, which is time-consuming. I think there is no better better way around.